08-data_exploration.Rmd

```diff
@@ -0,0 +1,145 @@
+# (PART\*) Data Exploration {-}
+
+
+```{r, include = FALSE}
+ottrpal::set_knitr_image_path()
+```
+# Instructor Guide
+
+# Student Guide
+
+Modules aimed at students in a course or workshop.
```

Comment:
I'm guessing we'll clean some of this up :)

Reply:
Yup! That is strictly there are a placeholder. I will clean it up when I find out what exactly we can have them do for data exploration (:

Reply:
Yeah, this is really just a placeholder for now.